docs/cli/index.mdx

```diff
@@ -138,39 +137,49 @@ Enable auto-completion using the following commands.
 
 ### zsh auto-completion
 
-<!-- TODO: add more information about zsh to make comparable to bash section -->
+1. Add the following line to your `~/.zshrc` file:
 
-Add the following code snippet to your `~/.zshrc` file:
+   ```sh
+   eval "$(temporal completion zsh)"
+   ```
 
-```sh
-source <(temporal completion zsh)
-```
+2. Re-launch your shell or run:
 
-If you're running auto-completion from the terminal, run the following command:
+   ```sh
+   source ~/.zshrc
+   ```
 
-```sh
-echo 'source <(temporal completion zsh)' >> ~/.zshrc
-```
+### Bash auto-completion
 
-After setting the variable, run the following command:
+1. Install [bash-completion](https://github.com/scop/bash-completion#installation) and add the software to your `~/.bashrc`.
 
-`source ~/.zshrc`.
+2. Add the following line to your `~/.bashrc` file:
 
-### Bash auto-completion
+   ```sh
+   eval "$(temporal completion bash)"
+   ```
 
-Install [bash-completion](https://github.com/scop/bash-completion#installation) and add the software to `~/.bash_profile`.
+3. Re-launch your shell or run:
 
-To use aliases, add them to `~/.bash_profile` as needed.
+   ```sh
+   source ~/.bashrc
+   ```
 
-```bash
-alias t='temporal'
-alias tw='temporal workflow'
-alias ts='temporal server start-dev'
-alias tsdb='temporal server start-dev --db-filename ~/temporal.db'
+### Fish auto-completion
 
-# send process to background so you can continue using the terminal
-alias tsbg='temporal server start-dev &> /dev/null & disown'
-```
+1. Make sure there is a place to store custom completions for Fish:
```

Comment:
A little more clarity that is action-oriented rather than "checking"

```suggestion
1. Create the Fish custom completions if one does not already exist:
```